JS/Lesson26/tugba/createPost/create.js

```diff
@@ -0,0 +1,33 @@
+
+const URL = 'https://jsonplaceholder.typicode.com/posts';
+
+const newForm = document.getElementById('post');
+
+async function createPost(e) {
+    e.preventDefault();
+    const title = document.getElementById('title');
+    const body = document.getElementById('body');
+
+    try {
+        const response = await fetch(URL, {
+            method: 'POST',
+            body: JSON.stringify({
+                title: Title.value,
```

Comment:
A small typo here: should be `title.value` as the variable declared above is called `title`